conda/deprecations.py

```diff
@@ -8,7 +8,7 @@
 
 from packaging.version import parse, Version
 
-from .__version__ import __version__
+from . import __version__
```

Comment:
IIRC `conda/__init__.py` imports `deprecations`, wouldn't that trigger a cyclic import?

Reply:
@kenodegard Can you take a look at this proposal please?

Reply:
even if this change works, why would we want it? it feels like its the opposite of what this PR is attempting to do

isn't it safer to import stuff from wherever it is defined instead of from other places where it was imported?